docs/editors/vim.md

```diff
@@ -323,7 +323,7 @@ Then, when on the line that you'd like to expand the decoration to get the hover
 
 coc-metals has a built-in implementation of the [Tree View
 Protocol](https://scalameta.org/metals/docs/editors/tree-view-protocol.html).
-If you have the [recommended mappings](coc-mappings.vim) copied, you'll notice
+If you have the [recommended mappings](vim.md#recommended-cocnvim-mappings) copied, you'll notice
```

Comment:
Just out of curiosity, since it's the same page this doesn't need to be prefaced with `vim.md` right?

Reply:
ach, right. I will remove it. I never remember the exact syntax needed, so I experiment a bit.

Reply:
Actually, I am getting a warning without it, so I will leave it as is.